src/MenuList.tsx

```diff
@@ -17,14 +16,18 @@ interface ListChildProps extends ListChildComponentProps {
   style: Style
 }
 
+interface OptionTypeBase {
```

Comment:
OptionTypeBase is removed from react-select. According to the upgrade docs:

> Options can be any type (if using getOptionValue and getOptionLabel) so there's no longer a base type for options

They've mapped it to unknown, but I pulled the old type back that's basically any object.